packages/module/src/CullingInfo/CullingInfo.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { Tooltip, TooltipProps } from '@patternfly/react-core';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+
+type Render = (config: { msg: string }) => React.ReactElement<any, any> | null;
+type CullingDate = string | number | Date;
+
+interface CullingInfo {
+  isWarn?: boolean;
+  isError?: boolean;
+  msg: string;
+}
+
+const seconds = 1000;
+const minutes: number = seconds * 60;
+const hours: number = minutes * 60;
+const days: number = hours * 24;
+
+type CalculateTooltip = (culled: CullingDate, warning: CullingDate, currDate: CullingDate) => CullingInfo;
+
+const useStyles = createUseStyles({
+  inventoryCullingWarning: {
+    color: 'var(--pf-t--global--icon--color--status--warning--default)',
+  },
+  inventoryCullingDanger: {
+    color: 'var(--pf-t--global--icon--color--status--danger--default)',
+  },
+  iconMargin: {
+    marginRight: 'var(--pf-t--global--spacer--sm)'
+  },
+  messageFont: {
+    fontWeight: 'var(--pf-t--global--font--weight--200)',
```

Comment:
rather than a base font weight token, let's use the semantic token `--pf-t--global--font--weight--body--bold`